### DISCUSSION TITLE
admin not a registered user

### DISCUSSION THREAD

**Discussion by bck-beep** [pH7Software/pH7-Social-Dating-CMS#1151] [Troubleshooting]
I installed ph7builder to the server through softaculous, can go to admin panel and set things etc, but for some reason I can not log into the actual site itself as a member,,,member 1 is usually automatically made as the admin on any other site i've seen or installed, but position 1 is not occupied in admin panel when browse members,,,so the question is why ? and how can I get the admin as a member, so can interact on the actual site and see it as members do,, hopefully someone can help with this, thank you in advance...

**Comment by Lumencraft** [comment]
same problem here..

**Comment by alleyvandal** [comment]
You have to just create a different account.
